### ISSUE TITLE
OperatorAdmin modification

### ISSUE THREAD

**Issue by jerolomb** [informed-governance-project/NISINP#177]
**Describe the bug**

- [x] Prevent to delete a user
- [x] allow to modify a user for the company concerned when a user is linked to several companies
- [ ] OperatorAdmin can't add new user under a sector in company

> Sachant que l'utilisateur est actif chez 2 opérateurs, et que les informations du deuxième opérateur ne sont pas renseignées pour des raisons de confidentialité, l'outil n'autorise pas la sauvegarde. NOK L'outil aurait dû autoriser la sauvegarde.

**Comment by jerolomb** [comment]
Actually an OperatorAdmin can add OperatorUser under sector he is belonging. If he has 0 sectors he can't add OperatorUser under a sector. 

Do you want to change this constraint by allowing an OperatorAdmin to add OperatorUser under any sector ? It implies that OperatorAdmin has access to all sector independently of its configuration in the administration page.

**Comment by PBD-ILR-LU** [comment]
NI : an operator, being he an OperatorAdmin or an OperatorUser, should be able to report an incident for any sector => no sector limitation

SO : the security objectives self-assessment is generic. It do not depend on any sector. => no sector limitation

RA : for the risk analysis the sector might be a relevant information, as it could be used to determine which sectoral library (ies) to user. Nevertheless, we plan to make these libraries public.

In other words, at this stage I do not see the need to have any sectoral limitation concerning the operators.

**Comment by jerolomb** [comment]
The question was more for GOV. 

I create an operatoradmin i give hime a sector, the operatoradmin can or not create an operatoruser under an other sector ?

**Comment by PBD-ILR-LU** [comment]
As mentioned, at this stage, I don't see the point of associating an operator with a sector. Therefore, I don't think we should prevent the operator admin from creating an operator user from another sector.
